Bottom: 1735763dbb389868e501f543994d4b4e373b8174
Top:    17e5231d99693ded1c97205738fc77cfccbfc966
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:18:02 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3a389db..ec4fafa 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1790,7 +1790,7 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
 	  poly_InclVertex (PREV_VERTEX (&(*result)->head), newn);
 	}
 #ifdef DEBUG_GATHER
-      DEBUGP ("gather vertex at %mm, %mm, Dir=%i\n", curv->point[0], curv->point[1], dir);
+      DEBUGP ("gather vertex at %$mn, %$mn, Dir=%i\n", curv->point[0], curv->point[1], dir);
 #endif
 
       /* Now mark the edge as included.  */
